src/main/java/com/keshavarzi/airplanestats/model/request/LoginRequest.java

```diff
@@ -0,0 +1,9 @@
+package com.keshavarzi.airplanestats.model.request;
+
+import lombok.Data;
+
+@Data
```

Comment:
Consider using `@Value` instead and for the other code in the CR as well, it generates the code to make the attributes of the class private AND final automatically.

Reply:
Refactored with Records